mopidy_mpd/session.py

```diff
@@ -21,6 +22,7 @@ def __init__(self, connection, config=None, core=None, uri_map=None):
         self.dispatcher = dispatcher.MpdDispatcher(
             session=self, config=config, core=core, uri_map=uri_map
         )
+        self.tagtypes = tagtype_list.TAGTYPE_LIST[:]
```

Comment:
```suggestion
        self.tagtypes = tagtype_list.TAGTYPE_LIST
```
And this would benefit from being a set rather than a list.